app/controllers/sprangular/locale_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Sprangular
+  ##
+  # API Endpoint to set locale for session
+  class LocaleController < Sprangular::BaseController
+    def set
```

Comment:
should be `def update`

in `config/routes.rb` we can use `resource :locale, only: :update`
to change locale, would be `$http.put('api/locale', locale: ...)`